examples/gitserver/README.md

```diff
@@ -38,6 +38,8 @@ Prerequisites:
     $ oc policy add-role-to-user edit -z git
     ```
 
+**NOTE:** To deploy the git server on OpenShift Online, you must switch to using persistent
+storage. See the *Persistent Storage* section below.
```

Comment:
@bparees had suggested that. But there's a couple of reasons that I don't like it but maybe you can convince me otherwise: 
- The PVC claim size has to be adjusted to your environment's claim size, which is 1G for online, but doesn't have to be that for other environments.
- The naming of things can get more complicated because now we'll have dc/git-ephemeral vs dc/git-persistent, etc. If possible I'd like it to remain short.